documentation/modules/exploit/windows/local/comahawk.md

```diff
@@ -0,0 +1,110 @@
+## Introduction
```

Comment:
I think it'd be good to mention that this is for Windows 10 and note the vulnerable build range.